### PR TITLE
Chore/6 base user service

### PR DESCRIPTION
**Pull request by Astralvix**
renamed branch and should be ready to go

### CODE REVIEW

File: src/main/java/com/revature/bean/Car.java [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+	@Column(name="seats")
+	@NotEmpty
+	@Positive
+	@Min(2)
+	@Max(50)
+	private int seatNumber;
+
+	public Car() {
+		super();
+		// TODO Auto-generated constructor stub
```
Comment (jurczakn) [inline src/main/java/com/revature/bean/Car.java]: Remove TODO comment

File: src/main/java/com/revature/bean/User.java [hunk 2]
```diff
@@ -0,0 +1,279 @@
...
+	@NotEmpty
+	UUID locationID;
+	
+	@JoinColumn(name="car_id")
+	@OneToOne
+	Car car;
+
+	public User() {
+		super();
+		// TODO Auto-generated constructor stub
```
Comment (jurczakn) [inline src/main/java/com/revature/bean/User.java]: TODO comment

File: src/main/java/com/revature/bean/User.java [hunk 3]
```diff
@@ -0,0 +1,279 @@
...
+	}
+
+	@Override
+	public String toString() {
+		return "User [userID=" + userID + ", email=" + email + ", firstName=" + firstName + ", lastName=" + lastName
+				+ ", phoneNumber=" + phoneNumber + ", rideStatus=" + rideStatus + ", role=" + role + ", accountStatus="
+				+ accountStatus + ", locationID=" + locationID + ", car=" + car + "]";
+	}
+
+
```
Comment (jurczakn) [inline src/main/java/com/revature/bean/User.java]: excess spacing

File: src/main/java/com/revature/bean/Car.java [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+			return false;
+		return true;
+	}
+
+	@Override
+	public String toString() {
+		return "Car [carID=" + carID + ", seatNumber=" + seatNumber + "]";
+	}
+	
+
```
Comment (jurczakn) [inline src/main/java/com/revature/bean/Car.java]: excess spacing

**Review by jurczakn — CHANGES_REQUESTED** [review]

File: src/main/java/com/revature/bean/Car.java (hunk 4 above)
Comment (Astralvix) [reply src/main/java/com/revature/bean/Car.java]: fixed

File: src/main/java/com/revature/bean/User.java (hunk 3 above)
Comment (Astralvix) [reply src/main/java/com/revature/bean/User.java]: fixed

File: src/main/java/com/revature/bean/User.java (hunk 2 above)
Comment (Astralvix) [reply src/main/java/com/revature/bean/User.java]: fixed

File: src/main/java/com/revature/bean/Car.java (hunk 1 above)
Comment (Astralvix) [reply src/main/java/com/revature/bean/Car.java]: fixed

File: src/main/java/com/revature/bean/User.java (hunk 2 above)
Comment (Astralvix) [reply src/main/java/com/revature/bean/User.java]: fixed

File: src/main/java/com/revature/bean/User.java (hunk 3 above)
Comment (Astralvix) [reply src/main/java/com/revature/bean/User.java]: fixed

File: src/main/java/com/revature/bean/Car.java (hunk 1 above)
Comment (Astralvix) [reply src/main/java/com/revature/bean/Car.java]: fixed

**Review by jurczakn — APPROVED** [review]
ok

**Pull request merged**